scheduler/pkg/store/pipeline/store.go

```diff
@@ -471,7 +471,7 @@ func (ps *PipelineStore) SetPipelineGwPipelineState(name string, versionNumber u
 	}
 	if ps.eventHub != nil {
 		for _, evt := range evts {
-			ps.eventHub.PublishPipelineEvent(setStatusPipelineEventSource, *evt)
+			ps.eventHub.PublishPipelineEvent(SetPipelineGwStatusPipelineEventSource, *evt)
```

Comment:
Could you please also fix the typo in the defintion: `pipeline.store.setpipelinegwtatus` -> `pipeline.store.setpipelinegwstatus`